tidb-in-kubernetes/tidb-backup-configuration.md

```diff
@@ -0,0 +1,107 @@
+---
+title: TiDB Backup 配置参数
+category: reference
+---
+
+# TiDB Backup 配置参数
+
+`TiDB-Backup` 是一个用于 TiDB 集群备份和恢复的 Helm Chart, 这篇文章将详细介绍 `TiDB-Backup` 的可配置参数。参考[备份与恢复](/dev/how-to/maintain/backup-and-restore.md)了解如何使用 `TiDB-Backup` 进行集群备份恢复。
+
+阅读本文档前，请参考 [Helm 使用说明](/TODO/to-helm-guide) 来了解如何修改这些配置项。 
+
+## `mode`
+
++ 运行模式
++ 默认： "backup"
++ 可选值为 `backup`(备份集群数据) 和 `restore`(恢复集群数据)
+
+## `clusterName`
+
++ 目标集群名字
++ 默认： "demo"
++ 指定要从哪个集群进行备份或将数据恢复到哪个集群中
+
+## `name`
+
++ 备份名
++ 默认值： "fullbackup-${date}"，`date` 是备份的开始时间，精确到分钟
++ 备份名用于区分不同的备份数据
+
+## `secretName`
+
++ 访问目标集群时使用的凭据
++ 默认： "backup-secret"
++ 该 Kubernetes Secret 中需要存储目标集群的登录用户名和密码，你可以通过以下命令来创建这个 Secret：
+    ```shell
+    $ kubectl create secret generic backup-secret -n ${namespace} --from-literal=user=root --from-literal=password=<password>
+    ```
+
+## `storage.className`
+
++ Kubernetes StorageClass 
++ 默认： "local-storage"
++ 备份任务需要绑定一个持久卷来永久或临时存储备份数据，`StorageClass` 用于声明持久卷使用的存储类型,，你需要确保该 `StorageClass` 在 Kubernetes 集群中存在。 
```

Comment:
Do you mean "Persistent Volume, PV" for "持久卷"? If yes, it would be better to provide the English name.

```suggestion
+ 备份任务需要绑定一个持久卷 (Persistent Volume, PV) 来永久或临时存储备份数据，`StorageClass` 用于声明持久卷使用的存储类型，需要确保该 `StorageClass` 在 Kubernetes 集群中存在。 
```